.env.local.example

```diff
@@ -1,6 +1,6 @@
 NEXT_PUBLIC_ENVIRONMENT=local
-NEXT_PUBLIC_CAPTCHA_KEY=
-NEXT_PUBLIC_API_BASE_URL=http://localhost:8080/graphql
+NEXT_PUBLIC_CAPTCHA_KEY=6Lca1skZAAAAAP808OdVHLjlzXRdiq4DH0HhdEqa
```

Comment:
I guess this must be publicly viewable anyway?

Reply:
Actually found a better alternative from [reCAPTCHA FAQ](https://developers.google.com/recaptcha/docs/faq) and changed to using it.